.github/workflows/sync-3rdparty-images.yaml

```diff
@@ -0,0 +1,104 @@
+name: 'Sync images and artifacts to ghcr'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions: read-all
+
+jobs:
+  sync-golang:
+    name: 'golang'
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        version:
+          - "1.20"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Tag and push to ghcr
+        run: |
+          docker trust inspect ${{ job.name }}:${{ matrix.version }}
+          docker pull ${{ job.name }}:${{ matrix.version }}
+          docker tag ${{ job.name }}:${{ matrix.version }} ghcr.io/${{ github.repository_owner }}/${{ job.name }}:${{ matrix.version }}
+          docker push ghcr.io/${{ github.repository_owner }}/${{ job.name }}:${{ matrix.version }}
```

Comment:
Can't we just use one of these run sections and multiple name:versions ?

Or just something like this:

```bash
for n in ubuntu:latest alpine:edge centos:latest; do
    dest="${{ github.repository_owner }}/$n"
    docker trust inspect "$n"
    docker pull $n
    docker tag $n "$dest"
    docker push $dest
done
```

Reply:
Fixed like so ^